app/models/difficulty_level.rb

```diff
@@ -3,5 +3,13 @@ class DifficultyLevel < ActiveRecord::Base
   has_many :events, dependent: :nullify
 
   validates :title, presence: true
-  validates :color, format: /\A#[0-9a-fA-F]{6}\z/
+  validates :color, format: /\A#[0-9A-F]{6}\z/
```

Comment:
You already have the validation added in 1ba59c2352925a1816a9f1113cf3749e8f869ac6 ?
I'd really like to see it in a separate commit! However, in this commit it makes more sense than in 1ba59c2352925a1816a9f1113cf3749e8f869ac6 ...